CONTENT_VS_CODE.md

```diff
@@ -0,0 +1,5 @@
+## Content vs. Code
+
+The **Website Team** (@nodejs/website) is primarily concerned with the code and overall structure of the website.
```

Comment:
Based on https://github.com/nodejs/nodejs.org/pull/5101#discussion_r1132339070 I think this should be edited

Reply:
This got edited already, and it's still true. We are primarily focused on the structure of the website, not the content.